### PR TITLE
Add feature_importance metric to SLR [solves #509]

### PR DESCRIPTION
**Pull request by ivanhigueram**
This commit adds a small change into the catwalk component to calculate feature importances when the model object is a `catwalk.estimators.ScaledLogisticRegression`. Now, instead of not
calculating anything, triage will be able to push  to the `feature_importances` db table the importances using the marginal change of the coefficients (using the exponential function). This solves the old #509.

### CODE REVIEW

File: src/triage/component/catwalk/feature_importances.py [hunk 1]
```diff
@@ -22,7 +23,8 @@ def _ad_hoc_feature_importances(model):
     """
     feature_importances = None
 
-    if isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)):
+    if (isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)) or
```
[the remark this answers is not in the document]
Comment (ivanhigueram) [reply src/triage/component/catwalk/feature_importances.py]: I guess for statistical reasons is better not to combine feature importances with the coefficients. The SLR it's estimating weights, rather than importances, and the calculation of the latter is mostly only to keep consistency across triage models. We're just rescaling and renaming marginal effects as feature importances here.

File: src/triage/component/catwalk/feature_importances.py [hunk 2]
```diff
@@ -22,7 +23,8 @@ def _ad_hoc_feature_importances(model):
     """
     feature_importances = None
 
-    if isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)):
+    if (isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)) or 
+        isinstance(model, (ScaledLogisticRegression))):
```
Comment (jesteria) [inline src/triage/component/catwalk/feature_importances.py]: no opinion on the more profound stuff, just FYI, this is briefer :wink: 

```
        if isinstance(model, (ScaledLogisticRegression,
                              sklearn.linear_model.logistic.LogisticRegression)):
```

Comment (ivanhigueram) [reply src/triage/component/catwalk/feature_importances.py]: 👍

File: src/triage/component/catwalk/feature_importances.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nanounanue) [reply src/triage/component/catwalk/feature_importances.py]: @thcrock What do you mean ? Like having a feature_importance attribute?

[the remark this answers is not in the document]
Comment (thcrock) [reply src/triage/component/catwalk/feature_importances.py]: @nanounanue yep, having the SLR implement the standard sklearn `.feature_importances_` attribute. You may not want to do it, but maybe you do.

[the remark this answers is not in the document]
Comment (ivanhigueram) [reply src/triage/component/catwalk/feature_importances.py]: Anything new here?
